packages/cosmic-swingset/src/anylogger-agd.js

```diff
@@ -54,10 +93,25 @@ anylogger.ext = logger => {
       // Enable the printing with a prefix.
       const doLog = l[level];
       if (doLog) {
+        const plainLevel = levelToPlainLevel[level];
         l[level] = (...args) => {
           // Add a timestamp.
-          const now = new Date().toISOString();
-          doLog(`${now} ${prefix}:`, ...args);
+          const now = new Date();
+          switch (logFormat) {
+            case 'plain': {
+              doLog(`${plainTime(now)} ${plainLevel} ${prefix}:`, ...args);
+              break;
+            }
+
+            case 'json':
+            default: {
+              const message = [`${prefix}:`, ...args]
+                .map(a => (typeof a === 'string' ? a : util.inspect(a)))
+                .join(' ');
+              doLog(JSON.stringify({ level, time: jsonTime(now), message }));
```

Comment:
Do we really want to flatten the message. The golang logger from what I recall does something a lot more natural where every other strings becomes keys. Maybe we could at least get a convention that a `log` call with even string values that are all suffixed with a `=` similarly create keys in the JSON output?